easybuild/easyconfigs/i/i7z/i7z-0.27.2-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'MakeCp'
+
+name = 'i7z'
+version = '0.27.2'
+
+homepage = 'https://github.com/ajaiantilal/i7z'
+description = """A better i7 (and now i3, i5) reporting tool for Linux
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+sources = [{
+    'filename': 'i7z-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/ajaiantilal',
+        'repo_name': 'i7z',
+        'commit': '5023138',
+        'recursive': True,
+        'keep_git_dir': True,
+    },
+}]
```

Comment:
Thanks for the suggestion, I have never contributed with easyconfigs before so have some learning to do :)
Just one question about how this works, does GitHub create a downloadable .tar.gz file for every commit?
Since if I understand your suggested change correctly we use `https://github.com/ajaiantilal/i7z` as the source url to pull the `5023138.tar.gz` source?

Reply:
whoops, that should be `source_urls = ['https://github.com/ajaiantilal/i7z/archive']`
and yes, github serves a tarball for each commit -- in this case, https://github.com/ajaiantilal/i7z/archive/5023138.tar.gz

Reply:
Okay cool, then I will change that!